src/imageattributes.c

```diff
@@ -337,8 +337,50 @@ GdipDisposeImageAttributes (GpImageAttributes *imageattr)
 }
 
 GpStatus WINGDIPAPI
+GdipSetImageAttributesToIdentity (GpImageAttributes *imageattr, ColorAdjustType type)
+{
+	GpImageAttribute *imgattr;
```

Comment:
OMG this variable name is so easy to confuse with the `imageattr` method argument. Not your fault of course since it's in the existing code too but maybe we should consider renaming these in the future :)